docs/AuthRBAC.md

```diff
@@ -553,614 +555,582 @@ export const PostEdit = () => (
+        // 'products.Section.stock' is missing
+    ]),
 };
 
-const ProductShow = () => (
-   <Show>
-       <TabbedShowLayout>
-           <Tab label="Description" name="description">
-               <TextField source="reference" />
-               <TextField source="width" />
-               <TextField source="height" />
-               {/* Not displayed */}
-               <TextField source="description" />
-           </Tab>
-           <Tab label="Images" name="images">
-               <TextField source="image" />
-               {/* Not displayed */}
-               <TextField source="thumbnail" />
-           </Tab>
-           {/* Not displayed */}
-           <Tab label="Stock" name="stock">
-               <TextField source="stock" />
-           </Tab>
-       </TabbedShowLayout>
-   </Show>
+const ProductEdit = () => (
+    <Edit>
+        <LongForm>
+            <LongForm.Section name="description" label="Description">
+                <TextInput source="reference" />
+                <TextInput source="width" />
+                <TextInput source="height" />
+                <TextInput source="description" />
+            </LongForm.Section>
+            <LongForm.Section name="images" label="Images">
+                <TextInput source="image" />
+                <TextInput source="thumbnail" />
+            </LongForm.Section>
+            <LongForm.Section name="stock" label="Stock">
+                <TextInput source="stock" />
+            </LongForm.Section>
+            // delete button not displayed
+        </LongForm>
+    </Edit>
 );
 ```
+{% endraw %}
 
-## Form Components
-
-### `<SimpleForm>`
-
-Alternative to react-admin's `<SimpleForm>` that adds RBAC control to inputs
+**Tip**: You must add a `name` prop to the `<LongForm.Section>` so you can reference it in the permissions.  
+Then, to allow users to access a particular `<LongForm.Section>`, update the permissions definition as follows: `{ action: 'write', resource: '{RESOURCE}.section.{NAME}' }`, where `RESOURCE` is the resource name, and `NAME` the name you provided to the `<LongForm.Section>`.
 
-To see an input, the user must have the permission to write the resource field:
+For instance, to allow users access to the following tab `<LongForm.Section label="Description" name="description">` in `products` resource, add this line in permissions: `{ action: 'write', resource: 'products.section.description' }`.
 
-```js
-{ action: "write", resource: `${resource}.${source}` }
-```
+`<LongForm.Section>` also only renders the child inputs for which the user has the 'write' permissions.
 
-`<SimpleForm>` also renders the delete button only if the user has the 'delete' permission.
+To learn more about the permissions format, please refer to the [`@react-admin/ra-rbac` documentation](https://marmelab.com/ra-enterprise/modules/ra-rbac).
 
-```jsx
-import { Edit, TextInput } from 'react-admin';
-import { SimpleForm } from '@react-admin/ra-rbac';
+{% raw %}
+```tsx
```

Comment:
It seems to me this the same example as above with some mistake:  
we talk about section but the comments talk about panel: `'products.panel.stock' is missing`.

To my opinion, we should add comments that explain which components is not displayed (`{/* not displayed */}`) in the above example and remove this example.